homeassistant/components/klyqa/config_flow.py

```diff
@@ -0,0 +1,307 @@
+"""Config flow for Klyqa."""
+from __future__ import annotations
+
+import asyncio
+from typing import Any
+
+from klyqa_ctl import klyqa_ctl as api
+from requests.exceptions import ConnectTimeout, HTTPError
+import voluptuous as vol
+
+from homeassistant import config_entries
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import (
+    CONF_HOST,
+    CONF_PASSWORD,
+    CONF_SCAN_INTERVAL,
+    CONF_USERNAME,
+)
+from homeassistant.data_entry_flow import FlowResult
+import homeassistant.helpers.config_validation as cv
+
+from .const import CONF_POLLING, CONF_SYNC_ROOMS, DOMAIN, LOGGER
+from .datacoordinator import HAKlyqaAccount
+
+user_step_data_schema = {
+    vol.Required(CONF_USERNAME, default=""): cv.string,
+    vol.Required(CONF_PASSWORD, default=""): cv.string,
+    vol.Required(CONF_SCAN_INTERVAL, default=60): int,
```

Comment:
This isn't allowed to be user configurable. Please remove it and provide a sane default out of the box.